project/assets/main/puzzle/levels/career/a-little-garbage.json

```diff
@@ -1,12 +1,19 @@
 {
   "version": "49db",
-  "start_speed": "1",
+  "start_speed": "4",
```

Comment:
Document these speed adjustments.

Reply:
Added commit message, `Adjusted a few level speeds to fit their corresponding career region.`